Update holidays.md (#358)
hyphacoop/handbook: How-we-work/holidays.md

# **Holidays, Time Off, and Leaves**

Hypha offers several types of time off, not including extended leaves (short/long term disability, parental leave, etc). This table describes the basics of how each type of time off functions, but more detail is given in each section below.

In addition to the public holidays and winter office closure, employees should generally be taking a **minimum of 3 weeks of vacation**, using up your Legislated Vacation Entitlement first, then your self-directed days off.

<table>
  <tr>
   <td><strong>Type</strong>
   </td>
   <td><strong>Amount</strong>
   </td>
   <td><strong>Flexibility</strong>
   </td>
   <td><strong>Approval and tracking</strong>
   </td>
  </tr>
  <tr>

>    <td><strong>Legislated Vacation Entitlement: </strong>This is the legislated minimum vacation entitlement and must be used by the end of the calendar year.  
>    </td>
>    <td>2 or 3 weeks, based on employment length and time commitment
>    </td>
>    <td>At employee discretion
>    </td>
>    <td>Inform your teammates and the Operations WG. Operations WG tracks vacation day usage for end-of-year payouts of unused days.
>    </td>
>   </tr>
>   <tr>
>    <td><strong>Self-Directed Vacation:</strong> These are additional paid vacation days outside of the ESA entitlement.
>    </td>
>    <td>Self-directed, available once Legislated Vacation Entitlement is used up
>    </td>
>    <td>At employee discretion, to be used responsibly given work and client commitments
>    </td>
>    <td>Managed on a per-project basis. The expectation is that they are generally approved and employees are considerate of client needs.
>    </td>
>   </tr>
>   <tr>
>    <td><strong>Lieu Days:</strong> These are paid days off in exchange for overtime work. 
>    </td>
>    <td>As needed (see section below)
>    </td>
>    <td>As project work allows
>    </td>
>    <td>Managed on a per-project basis.
>    </td>
>   </tr>
>   <tr>
>    <td><strong>Public Holidays: </strong>These are federally, provincially, and state legislated paid days off.
>    </td>
>    <td>12 days (9 Ontario public holidays + 3 additional holidays)
>    </td>
>    <td>Fixed (see section below)
>    </td>
>    <td>None
>    </td>
>   </tr>
>   <tr>
>    <td><strong>Winter Office Closure (Dec 27 - Dec 31 inclusive): </strong>These are paid days off in addition to the public holidays in December.
>    </td>
>    <td>3-5 days depending on weekends and whether Boxing Day is a public holiday in your province or state
>    </td>
>    <td>Fixed
>    </td>
>    <td>None
>    </td>
>   </tr>
>   <tr>
>    <td><strong>Wellness Days (including mental health days):</strong> These are paid days off for the purpose of health and wellness. 
>    </td>
>    <td>Self-directed
>    </td>
>    <td>As needed
>    </td>
>    <td>None, but employees must talk to their project lead about work planning after 6 consecutive sick days
>    </td>
>   </tr>
> </table>

Reviewer: I would suggest we use markdown over HTML for this table too
```suggestion
| Type | Amount | Flexibility | Approval and tracking |
|------|--------|-------------|----------------------|
| **Legislated Vacation Entitlement:** This is the legislated minimum vacation entitlement and must be used by the end of the calendar year. | 2 or 3 weeks, based on employment length and time commitment | At employee discretion | Inform your teammates and the Operations WG. Operations WG tracks vacation day usage for end-of-year payouts of unused days. |
| **Self-Directed Vacation:** These are additional paid vacation days outside of the ESA entitlement. | Self-directed, available once Legislated Vacation Entitlement is used up | At employee discretion, to be used responsibly given work and client commitments | Managed on a per-project basis. The expectation is that they are generally approved and employees are considerate of client needs. |
| **Lieu Days:** These are paid days off in exchange for overtime work. | As needed (see section below) | As project work allows | Managed on a per-project basis. |
| **Public Holidays:** These are federally, provincially, and state legislated paid days off. | 12 days (9 Ontario public holidays + 3 additional holidays) | Fixed (see section below) | None |
| **Winter Office Closure (Dec 27 - Dec 31 inclusive):** These are paid days off in addition to the public holidays in December. | 3-5 days depending on weekends and whether Boxing Day is a public holiday in your province or state | Fixed | None |
| **Wellness Days (including mental health days):** These are paid days off for the purpose of health and wellness. | Self-directed | As needed | None, but employees must talk to their project lead about work planning after 6 consecutive sick days |
```

## Legislated Vacation Entitlement

Hypha complies with Ontario’s [ESA Right to vacation - s. 33(1)](https://www.ontario.ca/document/employment-standard-act-policy-and-interpretation-manual/part-xi-vacation-pay#section-0):

*33(1) An employer shall give an employee a vacation of,*

1. *at least two weeks after each vacation entitlement year that the employee completes, if the employee’s period of employment is less than five years; or*
2. *at least three weeks after each vacation entitlement year that the employee completes, if the employee’s period of employment is five years or more.*

Employees receive a Legislated Vacation Entitlement of 2 or 3 weeks, depending on their period of employment. At Hypha, people typically consider a ‘week’ to be either 4 or 5 days.

The table below shows how you can check the number of days you are entitled to, based on your average days per work week and the number of weeks you are entitled to. If your average days per work week changes, this will be reflected in your Legislated Vacation Entitlement on a prorated basis.

> <table>
>   <tr>
>    <td><strong>Average days per work week</strong>
>    </td>
>    <td><strong>Legislated Vacation Entitlement in days (2 weeks) </strong>
>    </td>
>    <td><strong>Legislated Vacation Entitlement in days (3 weeks) </strong>
>    </td>
>   </tr>
>   <tr>
>    <td>1
>    </td>
>    <td>2
>    </td>
>    <td>3
>    </td>
>   </tr>
>   <tr>
>    <td>2
>    </td>
>    <td>4
>    </td>
>    <td>6
>    </td>
>   </tr>
>   <tr>
>    <td>3
>    </td>
>    <td>6
>    </td>
>    <td>9
>    </td>
>   </tr>
>   <tr>
>    <td>4
>    </td>
>    <td>8
>    </td>
>    <td>12
>    </td>
>   </tr>
>   <tr>
>    <td>5
>    </td>
>    <td>10
>    </td>
>    <td>15
>    </td>
>   </tr>
> </table>

Reviewer: I suggest we use markdown tables over html elements

```suggestion
| Average days per work week | Legislated Vacation Entitlement in days (2 weeks) | Legislated Vacation Entitlement in days (3 weeks) |
|----------------------------|---------------------------------------------------|---------------------------------------------------|
| 1                          | 2                                                 | 3                                                 |
| 2                          | 4                                                 | 6                                                 |
| 3                          | 6                                                 | 9                                                 |
| 4                          | 8                                                 | 12                                                |
| 5                          | 10                                                | 15                                                |
```

Hypha schedules the winter closure (3-5 days), which can be deducted from an employee’s remaining Legislated Vacation Entitlement at the end of the year to reduce the need to calculate and pay out unused days.

For example, in a year where the winter closure is 5 days, an employee ending the year with &lt;=5 unused Legislated Vacation Entitlement days will not receive a payout, as the winter closure is considered to be vacation scheduled by the employer. 

If an employee uses up the entirety of their Legislated Vacation Entitlement, they are still expected to observe the winter closure - it’s free vacation :) Please use up all your vacation days; it is in everyone’s best interest!

### How to track vacation time

1. Inform your relevant coworkers (project lead, project teammates, working group) of your intended vacation.
2. Send an email to Operations with your vacation start and end dates, and the number of days you will be taking off. If you work &lt;5 days per week, your usual day off should not be tracked as a vacation day (e.g.: if you don’t work Fridays in a regular work week, you are not taking Friday as a vacation day)

> 3. Operations will update the[ vacation tracking spreadsheet](https://link.hypha.coop/vacation)🔒

Reviewer: ```suggestion
3. Operations will update the [vacation tracking spreadsheet](https://link.hypha.coop/vacation)🔒
```

## Lieu Days

Hypha employees operate on a salary basis, which means we are paid a fixed amount for the work we contribute. A salaried position implies some give-and-take when it comes to hourly labour – some amount of underwork and overwork is expected on a per-week basis. Generally, this evens out, but employees should keep track of their own time commitment (e.g., 4 days of work per week) and inform their project lead when they are notably under- or over-worked (e.g., working 3 or 5 days on a 4 day commitment).

When a member works beyond the allotted days in their full work week, they are expected to take a lieu day at a later date so as not to overwork themself. For example, if Alice commits to working a four-day work week but ends up working five days in a particular week, they should work three days the next week (or within a reasonable amount of time).

Lieu days are self-tracked and arranged with the member's practice leads so as not to be disruptive to the project.

## Public and Additional Holidays + Winter Closure

> Our virtual office is closed during[ Ontario's nine public holidays](https://www.ontario.ca/document/your-guide-employment-standards-act-0/public-holidays). We recognize three additional paid holidays where the office is closed and employees are expected not to work:

Reviewer: ```suggestion
Our virtual office is closed during [Ontario's nine public holidays](https://www.ontario.ca/document/your-guide-employment-standards-act-0/public-holidays). We recognize three additional paid holidays where the office is closed and employees are expected not to work:
```

* Civic Holiday (first Monday in August)
* May Day or International Workers' Day (May 1)
* Remembrance Day (November 11)

> If you work outside of Ontario, you will observe your province or state’s public holidays, along with the additional paid holidays listed above. 

Reviewer: Really? 
TIL I am observing St-Jean-Baptiste tomorrow  !
